bindings/go/src/fdb/transaction.go

```diff
@@ -184,7 +184,9 @@ func (t Transaction) Snapshot() Snapshot {
 // Typical code will not use OnError directly. (Database).Transact uses
 // OnError internally to implement a correct retry loop.
 func (t Transaction) OnError(e Error) FutureNil {
-	return &futureNil{newFuture(C.fdb_transaction_on_error(t.ptr, C.fdb_error_t(e.Code)))}
```

Comment:
Are the changes to this file just whitespace changes?

Reply:
Yea, although I accidentally deleted some of the labels which I added back.

I also added labels to the structs with just one field even though it isn't necessary to make everything uniform.